.github/workflows/benchmark-labeled.yml

```diff
@@ -9,7 +9,92 @@ on: # TODO: delete this file when specifying a label is supported
       - '*noci'
 
 jobs:
-  Invoker:
-    if: github.event.label.name == 'benchmark'
+  #runs basics fp 32 single + multi and spaces
+  default_invoker: 
+    if: github.event.label.name == 'benchmarks-default'
     uses: ./.github/workflows/benchmark.yml
     secrets: inherit
+
+  # basics benchmarks labels
+  basics_fp32_s_invoker:
+    if: github.event.label.name == 'bm basics fp32 single'
+    uses: ./.github/workflows/benchmark.yml
+    with:
+      filter: "BASICS_FP32_S"
```

Comment:
can we try to use `fromjson` https://stackoverflow.com/questions/60225843/map-an-environment-variable-in-github-actions
so the json will be a map between the label to the right filter, and have only one invoker here?